simulation/Inverted Pendulum.ipynb

```diff
@@ -0,0 +1,133 @@
+{
```

Comment:
I could not get the animation to work. I could get a static image by replacing `%matplotlib notebook` with `%matplotlib inline`. Are you port forwarding this and opening within your browser?

Reply:
running 'jupyter notebook' from linux command line, which opens a chrome browser instance

Reply:
Cool. Was able to reproduce within the dev container.